### PR TITLE
Update install devTools

### PR DESCRIPTION
**Pull request by laurentheirendt**
**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

### CODE REVIEW

File: src/base/install/installDevTools.m [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+function installDevTools(repoName)
 % checks if the MATLAB.devTools are already present and installs the
 % MATLAB.devTools next to the cloned folder cobratoolbox (if not present yet)
 %
 % USAGE:
-%    installDevTools()
+%    installDevTools(repoName)
+%
+% INPUT:
+%    repoName:     Name of the repository (default: 'opencobra/cobratoolbox')
```
Comment (tpfau) [inline src/base/install/installDevTools.m]: If there is a default, then this is an optional input.

File: README.rst [hunk 2]
```diff
@@ -111,7 +111,25 @@ within MATLAB by typing:
 
     >> installDevTools()
 
-|bulb| Check out the `MATLAB.devTools <https://github.com/opencobra/MATLAB.devTools>`__ - and contribute the smart way!
+|bulb| Check out the `MATLAB.devTools
+<https://github.com/opencobra/MATLAB.devTools>`__ - and contribute the smart
+way! The **official documentation** is `here <https://opencobra.github.com/MATLAB.devTools>`__.
```
Comment (tpfau) [inline README.rst]: this link <https://opencobra.github.com/MATLAB.devTools> (at least at the moment) does not work. 
I guess it should be:
in any case this should be (I think) https://opencobra.github.io/MATLAB.devTools, but even that link does not work.

File: src/base/install/installDevTools.m [hunk 3]
```diff
@@ -99,7 +106,7 @@ function installDevTools()
             % removing .git folder
             rmpath(genpath([devToolsDir filesep '.git']));
 
-            fprintf([' > In order to start contributing, you can now run:\n\n >> contribute\n\n']);
+            fprintf([' > In order to start contributing, you can now run:\n\n >> contribute(''' repoName '''); \n\n']);
```
Comment (tpfau) [inline src/base/install/installDevTools.m]: One question:
Will it be necessary to always indicate the repoName when using contribute? Or will this (by default) be detected automatically based on the git information in the current folder?

File: src/base/install/installDevTools.m (hunk 1 above)
Comment (laurentheirendt) [reply src/base/install/installDevTools.m]: True.

File: README.rst (hunk 2 above)
Comment (laurentheirendt) [reply README.rst]: Once the regular merge on the MATLAB.devTools has been made, these links will work. We are aware of that.

File: src/base/install/installDevTools.m (hunk 3 above)
Comment (laurentheirendt) [reply src/base/install/installDevTools.m]: It is defined as follows:
```
>> contribute
```
is the same as
```
>> contribute('opencobra/cobratoolbox')
```
When contributing to any other repo:
```
>> contribute('orgName/repoName')
```

**Comment by syarra** [comment]
Great! Thanks @laurentheirendt 🌵

**Pull request merged**